### PR TITLE
feat: bump hubble-sdk  v0.22.2, show local location of Executors in Hub

### PR DESCRIPTION
**Pull request by floralatin**
**Goals:**

Add `jina hub list` command for showing the local location of Jina executors.

[asciicast](https://asciinema.org/a/z81wi9gwVm7gYjfl5ocBD1RH3)

 
- [x] check and update documentation. See [guide](https://github.com/jina-ai/jina/blob/master/CONTRIBUTING.md#-contributing-documentation) and ask the team.

### CODE REVIEW

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 1]
```diff
@@ -153,3 +153,11 @@ jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
 
 
 The source code of the Executor will be stored at `~/.jina/hub-packages`.
+
+### list the local location of executors
+
+```bash
+jina hub list
```
Comment (JoanFM) [inline docs/fundamentals/executor/hub/use-hub-executor.md:160]: can u please add an output example?

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 2]
```diff
@@ -153,3 +153,11 @@ jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
 
 
 The source code of the Executor will be stored at `~/.jina/hub-packages`.
+
+### list the local location of executors
```
Comment (alexcg1) [inline docs/fundamentals/executor/hub/use-hub-executor.md:157]: For headings, ensure:

- Top level headings (`# foo`) - [Title Case](https://titlecaseconverter.com/#TitleCapitalizationRules)
- Lower level headings (`## bar`, `### baz`) - [Sentence case](https://titlecaseconverter.com/#WhatIsSentenceCase)

Links above go to a tool that will auto-format your text accordingly.

We use AP style

Comment (alexcg1) [inline docs/fundamentals/executor/hub/use-hub-executor.md:157]:
```suggestion
### List locations of local Executors
```

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 3]
```diff
@@ -153,3 +153,11 @@ jina hub pull jinahub://<UUID>[:<SECRET>][/<TAG>]
...
+
+```bash
+jina hub list
+```
+
+Show the local location of Jina executors.
```
Comment (alexcg1) [inline docs/fundamentals/executor/hub/use-hub-executor.md:163]: This is just rewording of the title. We need something better here, or just remove it altogether

**Review by alexcg1 — CHANGES_REQUESTED** [review]

File: docs/fundamentals/executor/hub/use-hub-executor.md (hunk 1 above)
Comment (floralatin) [reply docs/fundamentals/executor/hub/use-hub-executor.md:160]: We'll record a command line video about it

File: docs/fundamentals/executor/hub/use-hub-executor.md (hunk 2 above)
Comment (floralatin) [reply docs/fundamentals/executor/hub/use-hub-executor.md:157]: Thanks, i will convert  `list the local location of executors`  to` List locations of local Executors`

File: docs/fundamentals/executor/hub/use-hub-executor.md (hunk 3 above)
Comment (floralatin) [reply docs/fundamentals/executor/hub/use-hub-executor.md:163]: I will remove it and add the command line video.

File: docs/fundamentals/executor/hub/use-hub-executor.md (hunk 2 above)
Comment (floralatin) [reply docs/fundamentals/executor/hub/use-hub-executor.md:157]: Thanks for the suggestion, I will fix it.

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 4]
```diff
@@ -151,11 +151,16 @@ jina hub pull jinahub+docker://DummyExecutor/v1.0.0
...
 ### List locations of local Executors
 
 ```bash
 jina hub list
 ```
+
+<script id="asciicast-z81wi9gwVm7gYjfl5ocBD1RH3" src="https://asciinema.org/a/z81wi9gwVm7gYjfl5ocBD1RH3.js" async></script>
+
+```{tip}
+`jina hub list` command only lists all the local Executors with source code format.
```
Comment (alexcg1) [inline docs/fundamentals/executor/hub/use-hub-executor.md:163]:
```suggestion
To list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list`
```

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 5]
```diff
@@ -151,11 +151,16 @@ jina hub pull jinahub+docker://DummyExecutor/v1.0.0
...
 ```bash
 jina hub list
 ```
+
+<script id="asciicast-z81wi9gwVm7gYjfl5ocBD1RH3" src="https://asciinema.org/a/z81wi9gwVm7gYjfl5ocBD1RH3.js" async></script>
+
+```{tip}
+`jina hub list` command only lists all the local Executors with source code format. 
+
+If you want to list all the docker format Executors, you can explictly use `docker images`.
```
Comment (alexcg1) [inline docs/fundamentals/executor/hub/use-hub-executor.md:165]:
```suggestion
To list all the Executors that are in Docker format, use the command `docker images`. 
```

Be sure to check spelling in future!

**Review by alexcg1 — CHANGES_REQUESTED** [review]

File: docs/fundamentals/executor/hub/use-hub-executor.md (hunk 4 above)
Comment (floralatin) [reply docs/fundamentals/executor/hub/use-hub-executor.md]: I will convert  ``jina hub list` command only lists all the local Executors with source code format.`   to  T`o list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list``

File: docs/fundamentals/executor/hub/use-hub-executor.md [hunk 6]
```diff
@@ -151,5 +151,16 @@ jina hub pull jinahub+docker://DummyExecutor/v1.0.0
...
 
-The source code of the Executor will be stored at `~/.jina/hub-packages`.
+```bash
+jina hub list
+```
+
+<script id="asciicast-z81wi9gwVm7gYjfl5ocBD1RH3" src="https://asciinema.org/a/z81wi9gwVm7gYjfl5ocBD1RH3.js" async></script>
+
+```{tip}
+To list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list`
```
Comment (alexcg1) [inline docs/fundamentals/executor/hub/use-hub-executor.md:163]:
```suggestion
To list all the Executors that are in source-code format (i.e. pulled via `jinahub://`), use the command `jina hub list`.
```

**Review by alexcg1 — CHANGES_REQUESTED** [review]

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:

**Comment by floralatin** [comment]
@JoanFM  Could you help me to merge it, it has been approved.

**Review by JoanFM — APPROVED** [review]

**Pull request merged**